### PR TITLE
Update index.jsp

### PR DESCRIPTION
**Pull request by Paul-Kalil**
Replace wild dog profile image with wolf profile image

WNWB_wolf_lowres

### CODE REVIEW

File: src/main/webapp/index.jsp [hunk 1]
```diff
@@ -286,7 +286,7 @@ h2.vidcap {
 	                           Encounter thisEnc=latestIndividuals.get(i);
 	                           %>
 	                            <li>
-	                                <img src="images/wild_dog_logo_2.png" alt="" width="85px" height="75px" class="pull-left" />
+	                                <img src="images/WNWB_wolf_lowres.png" alt="" width="85px" height="75px" class="pull-left" />
```
Comment (naknomum) [inline src/main/webapp/index.jsp:289]: think this `WNWB_wolf_lowres.png` image also needs to be added to this PR as well.

**Review by naknomum — CHANGES_REQUESTED** [review]
just a missing image needed, i believe.

**Comment by Paul-Kalil** [comment]
Thanks

Please go ahead.

Paul

**Comment by naknomum** [comment]
> Hi Jon I thought I had attached it, apologies. Here it is again. Thanks Paul

oh i think its just a misunderstanding. the file itself needs to be added to your `patch-18` branch, underneath the `src/main/webapp/images/` directory.  then do some git commands to include it, something like:

```
git add src/main/webapp/images/WNWB_wolf_lowres.png
git commit -m 'new logo added' src/main/webapp/images/WNWB_wolf_lowres.png
git push
```
hopefully i have that about right. then the new file should show up here on this PR, in addition to the change you made to `index.jsp`.

**Comment by naknomum** [comment]
@Paul-Kalil - did another PR or `master` leak into this? it went from the one file (`index.jsp`) being changed to having these others: `RowFeedback.java`, `dependabot.yml`, and so on.

i am just a little confused as to what might have happened here. :thinking:

**Comment by Paul-Kalil** [comment]
Hi Jon

I synced with the Master and all these updates came across.

Can we just ignore them?  And did I succeed in getting the new image
uploaded to Patch-18?

I had trouble with the Command Line approach so ended up getting the GitHub
desktop app.

Thanks

Paul

**Comment by naknomum** [comment]
> Hi Jon I synced with the Master and all these updates came across. Can we just ignore them? And did I succeed in getting the new image uploaded to Patch-18? I had trouble with the Command Line approach so ended up getting the GitHub desktop app. Thanks Paul

yeah i have seen where this can happen - git command line and gui tools can get quite complicated with multiple branches, especially since master is getting worked on so frequently. to be honest, i think _undoing_ this might be trickier than starting fresh with a new PR directly from (github's) `wildnorth` branch. this will give us a simpler merge path for `wildnorth`, i think. (then the master changes can come in after.)

i can do that really quickly, since you _did_ get the image and change to index.jsp in here completely now. i will leave a comment here when that is up, so you can review it and make sure i didnt miss your changes.

**Comment by naknomum** [comment]
@Paul-Kalil please take a look at [PR456](https://github.com/WildMeOrg/Wildbook/pull/456). i think this has just the 2 modifications you made. if that looks good we can get that into `wildnorth` first, then bringing over changes from master can follow.

**Comment by naknomum** [comment]
closing this as it was superceded by [PR456](https://github.com/WildMeOrg/Wildbook/pull/456).

**Pull request closed without merging**